extensions/sql-database-projects/package.json

```diff
@@ -293,6 +319,11 @@
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.project && azdataAvailable",
           "group": "1_dbProjectsFirst@3"
         },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+          "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.project",
```

Comment:
do we need to hide this command in the vscode extension because there's no schema compare in vscode?